src/Symfony/Component/Stopwatch/StopwatchPeriod.php

```diff
@@ -25,20 +25,20 @@ class StopwatchPeriod
     /**
      * Constructor.
      *
-     * @param int $start The relative time of the start of the period (in milliseconds)
-     * @param int $end   The relative time of the end of the period (in milliseconds)
+     * @param float $start The relative time of the start of the period (in milliseconds)
+     * @param float $end   The relative time of the end of the period (in milliseconds)
      */
     public function __construct($start, $end)
     {
-        $this->start = (int) $start;
-        $this->end = (int) $end;
+        $this->start = $start;
+        $this->end = $end;
         $this->memory = memory_get_usage(true);
     }
 
     /**
      * Gets the relative time of the start of the period.
      *
-     * @return int The time (in milliseconds)
+     * @return float The time (in milliseconds)
      */
     public function getStartTime()
```

Comment:
changing output of this is BC breaker, and for one that like to write in new php in strict mode, it will crash his software.

```php
<?php

declare(strict_types=1);

function NON_STRICTLY_TYPEHINTED_SYMFONY_METHOD___getStartTime_org() { return 123; }

function NON_STRICTLY_TYPEHINTED_SYMFONY_METHOD___getStartTime_new() { return 123.456; }

function USER_FUNCTION_THAT_CONSUMES_SYMFONY_OUTPUT(int $a) {
    echo $a;
}

USER_FUNCTION_THAT_CONSUMES_SYMFONY_OUTPUT(1);   // OK
USER_FUNCTION_THAT_CONSUMES_SYMFONY_OUTPUT(1.5); // Fatal error: Uncaught TypeError: Argument 1 passed to USER_FUNCTION_THAT_CONSUMES_SYMFONY_OUTPUT() must be of the type integer, float given
```